app/assets/stylesheets/application.css.scss

```diff
@@ -16,76 +16,77 @@
 
  */
 @import 'foundation-icons';
+
+/* CUSTOM VARIABLES */
+
 $boost-red: #B72039;
 
+/* FOUNDATION BREAKPOINTS */
+
+$medium-start: 40.063em;
```

Comment:
Could replace these with the ones from the foundation overrides file at some point

Reply:
Done in next commit :)